### PR TITLE
fix: catch error

### PR DESCRIPTION
**Pull request by rottenpen**
#62

### CODE REVIEW

**Comment by rottenpen** [comment]
cc @ije

**Comment by rottenpen** [comment]
@roj1512 @ije PTAL

File: blog.tsx [hunk 1]
```diff
@@ -474,8 +482,14 @@ export function redirects(redirectMap: Record<string, string>): BlogMiddleware {
         },
       });
     }
-
-    return await ctx.next();
+    try {      
+      return await ctx.next();
+    } catch (e) {
+      console.log(e)
```
Comment (ije) [inline blog.tsx:488]: please use `console.error`

**Review by ije — COMMENTED** [review]
Looks great, can you please update the `console.error`? thanks

**Comment by rottenpen** [comment]
> Looks great, can you please update the `console.error`? thanks

done~

**Comment by ije** [comment]
LGTM

**Comment by ije** [comment]
please run `deno fmt`

**Pull request merged**